sunpy/io/tests/test_fits.py

```diff
@@ -176,3 +177,11 @@ def test_warn_longkey():
                                'goodkey': 'test'})
     assert 'GOODKEY' in fits.keys()
     assert 'BADLONGKEY' not in fits.keys()
+
+
+def test_read_memmap():
+    # Check that memmap is read correctly from a file
+    data, header = sunpy.io._fits.read(TEST_AIA_IMAGE, memmap=True)[0]
+    if data.base is None:
+        assert False
+    assert isinstance(data.base, mmap.mmap) or isinstance(data.base, np.memmap)
```

Comment:
Do we need both asserts and the `or`?

Reply:
We don't need to check `isinstance(data.base, np.memmap)`.  `astropy.io.fits.open()` uses only `mmap.mmap`, so this unit test doesn't need to anticipate a possible future where `np.memmap` might be used.

Reply:
Sure